manager/src/main/resources/application.properties

```diff
@@ -38,7 +38,8 @@ quarkus.flyway.migrate-at-start=true
 quarkus.flyway.validate-on-migrate=true
 quarkus.flyway.ignore-missing-migrations=true
 quarkus.flyway.create-schemas=true
-quarkus.flyway.placeholders.shard-id=${SHARD_ID}
+quarkus.flyway.placeholders.shard-id=${SHARD_ID:150b23e9-0b34-45b7-a91a-5ec388d03a1d}
```

Comment:
Is there any reason to start to provide default SHARD-ID here?

Reply:
As it is not mandatory I've removed it